### PR TITLE
Add Coast Capital

### PR DESCRIPTION
**Pull request by daylenca**

### CODE REVIEW

**Comment by Carlgo11** [comment]
>We’ll send a one-time security code via text message to your mobile phone or through an email when you complete specific tasks.

That sounds like [authorization](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#authorization) rather than authentication.

**Comment by daylenca** [comment]
Thanks, @Carlgo11! I've updated the json file in commit #86d01dd to reflect that 2FA is not available.

File: img/s/scotiabank.com.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80.7 89.5"><path fill="#ec121f" d="M65.7 16.7H40.4a28 28 0 0 0-28 25A28 28 0 0 1 37 0h43.7ZM15 72.8h25.4a28 28 0 0 0 27.9-25 28 28 0 0 1-24.6 41.7H0Z" class="cls-1"/><circle cx="40.4" cy="44.7" r="22.5" fill="#ec121f" class="cls-1"/></svg>
```
Comment (Carlgo11) [inline img/s/scotiabank.com.svg:1]: Why is this file here?
